t/units/conversions.t

```diff
@@ -0,0 +1,143 @@
+	is multiply_by(1000, evaluate_units('ohm')), { evaluate_units('kohm') }, 'kilo-ohm conversion';
+	is multiply_by(1E6,  evaluate_units('ohm')), { evaluate_units('Mohm') }, 'kilo-ohm conversion';
+	is multiply_by(1000, evaluate_units('mV')),  { evaluate_units('V') },    'millivolt conversion';
+	is multiply_by(1000, evaluate_units('V')),   { evaluate_units('kV') },   'kilovolt conversion';
+};
+
+subtest 'Check magnetic units' => sub {
+	is multiply_by(1e4, evaluate_units('G')), { evaluate_units('T') }, 'magnetic field strength conversion';
+	is { evaluate_units('V/ohm') }, { evaluate_units('V*S') },     'conductivity definition';
+	is { evaluate_units('Wb') },    { evaluate_units('T*m^2') },   'Weber definition';
+	is { evaluate_units('H') },     { evaluate_units('V*s/amp') }, 'Henry definition';
+};
+
+subtest 'Check biological and chemical units' => sub {
+	is multiply_by(1000, evaluate_units('micromol/L')), { evaluate_units('mmol/L') }, 'concentration conversion';
+	is multiply_by(10,   evaluate_units('mg/L')),       { evaluate_units('mg/dL') },  'concentration conversion';
+	is multiply_by(1e9,  evaluate_units('nanomol')),    { evaluate_units('mol') },    'concentration conversion';
+};
+
+subtest 'Check radiation units' => sub {
+	is multiply_by(1000, evaluate_units('mSv')), { evaluate_units('Sv') }, 'milli-Sievert conversion';
+	is multiply_by(1e6,  evaluate_units('uSv')), { evaluate_units('Sv') }, 'micro-Sievert conversion';
+	is { evaluate_units('kat') }, { evaluate_units('mol/s') }, 'catalitic activity';
+};
+
+subtest 'Check a collection of units' => sub {
+	is multiply_by(1822.88854680448, evaluate_units('me')), { evaluate_units('amu') }, 'atomic mass conversion';
+
+	is { evaluate_units('lx') }, { evaluate_units('lm/m^2') }, 'lux = lumen per square metre';
+
+	is multiply_by(1e9,  evaluate_units('Pa')),  { evaluate_units('GPa') }, 'gigapascal conversion';
+	is multiply_by(1000, evaluate_units('kPa')), { evaluate_units('MPa') }, 'kilopascal conversion';
+
+	is multiply_by(2 * 1000 * $Units::PI, evaluate_units('rad/s')), { evaluate_units('kHz') },
+		'kilohertz conversion';
+
+	is multiply_by(0.01, %Units::fundamental_units), { evaluate_units('%') }, 'percent conversion';
+
+	my $todo = todo 'use within() to fudge factor in 9th decimal place';
+	is multiply_by((180 / $Units::PI)**2, evaluate_units('deg^2')), { evaluate_units('sr') },
+		'solid angle conversion';
+};
+
+subtest 'Check astronomical units' => sub {
+	my $second_arc = 0.0174532925 / 60 / 60;
+
+	is multiply_by(299792458, evaluate_units('m/s')), { evaluate_units('c') }, 'speed of light conversion';
+	is { evaluate_units('c*yr') }, { evaluate_units('light-year') }, 'light year';
+
+	my $todo = todo 'use within() to fudge factor in 9th decimal place';
+	is multiply_by(cos($second_arc) / sin($second_arc), evaluate_units('AU')), { evaluate_units('parsec') },
+		'parsec conversion';
+};
+
+subtest 'Recently added electrical units' => sub {
```

Comment:
Using "Recently" will seem strange when running this test in a couple of years.  Perhaps "Additional electrical units"